### ISSUE TITLE
Playback Options (force HLS)

### ISSUE THREAD

**Issue by nichwall** [advplyr/audiobookshelf#2040]
### Describe the feature/enhancement

As discussed in Discord, there should be a "Force Transcode" option for playing a file. This would help solve the problem of certain files not playing because the client incorrectly reports the codec is supported.

Two ideas I have for UI
- Add a small green box drop-down on the right of the play button (about same size as blue Read box for epub) but opens a drop-down to select force transcode for the playback, and would provide a place for other playback options (such as continue series? Not sure what else)
- Add a checkbox/button to the Player options to force a transcode to HLS. I'm not sure how that would work if a user tries to enable/disable it, so maybe don't allow transcode to be disabled until the player is closed

**Comment by nichwall** [comment]
Yes, some just talked about it yesterday on Discord (Firefox). Exor has also mentioned the transcode not happening automatically on Firefox recently but can't find that conversation.

Initial question and discussion from yesterday (there were three conversations going on at the same time) https://discord.com/channels/942908292873723984/942914154254176257/1146372596003242074

**Comment by nichwall** [comment]
Does this PR account for "all" of the other stream errors?

https://github.com/advplyr/audiobookshelf/pull/2171

**Comment by advplyr** [comment]
It wouldn't handle that specific issue from the discord chat because the client isn't throwing an error, it's just not playing anything.
